src/main/python/api.py

```diff
@@ -1,44 +1,45 @@
 from os import environ, path, curdir, remove
 import requests as req
 
-from json import JSONDecodeError
+from json import loads, dumps, JSONDecodeError
 from requests.exceptions import ConnectionError
 
 from fbs_runtime.application_context.PyQt5 import ApplicationContext
 
 
 class CredentialManager(object):
     # TODO: Consider using something like marshal, shelve, or pickle
-    def __init__(self, cxt: ApplicationContext, file_path="./"):
+    def __init__(self, cxt: ApplicationContext):
 
-        self.credential_store = cxt.credential_store
-        # self.file_path = file_path
-        # if not path.exists(path.join(path.abspath(self.file_path), ".credential_store")):
-        #     self.credential_store = path.join(path.abspath(self.file_path), ".credential_store")
+        self.store_path = cxt.credential_store
+        with open(self.store_path, "r") as store:
+            self.store = loads(dumps(store.read()))
```

Comment:
any reason to do this rather than just `dict()`?  Seems like we are reading a dict, dumping into a json, then loading into a dict again?

Reply:
or can `load` return something other than a dict

Reply:
ah nevermind looks like this can be many different kinds of datatypes: https://docs.python.org/3/library/json.html#json-to-py-table

disregard

Reply:
actually maybe since we control what's written in store, it looks like the only place that it could be written to not be a dict is when setting `self.store = "store"`.  Why not set that to `{ 'token': 'invalid_token' }` or some other special value to avoid the loads(dumps(reads())) call?

Reply:
It should hopefully always be a `dict` literal that's stored in the file (for that matter, the decode/encode logic should probably be wrapped in a `try`/`catch`). I'm not entirely sure why, but this was the only way I was able to properly parse the file. I realize it's ugly but I think the work to fix it should be handled in another PR that implements something a little closer to a database rather than this hastily put together text file.